scripts/yt-api-helper.sh

```diff
@@ -0,0 +1,410 @@
+#!/bin/sh
+# shellcheck disable=SC2236
```

Comment:
I am fine with ignoring this, just why?

Reply:
For two reasons: 1) as far as I know `! [ -z $var ]` is supported by all shells on all platforms, where as `[ -n $var ]` is not supported by some non-POSIX compliant shells and 2) It's easier to understand imo.